frame/staking/src/lib.rs

```diff
@@ -2788,25 +3018,27 @@ impl<T: Config> frame_election_provider_support::ElectionDataProvider<T::Account
 		// NOTE: reading these counts already needs to iterate a lot of storage keys, but they get
 		// cached. This is okay for the case of `Ok(_)`, but bad for `Err(_)`, as the trait does not
 		// report weight in failures.
```

Comment:
```suggestion
```
no longer relevant? (expect for the `iter` on `SlashingSpans` to get `slashing_span_count`)

Reply:
True.